.github/release-drafter.yml

```diff
@@ -1,5 +1,5 @@
-name-template: '$RESOLVED_VERSION'
-tag-template: '$RESOLVED_VERSION'
+name-template: 'v$RESOLVED_VERSION'
+tag-template: 'v$RESOLVED_VERSION'
```

Comment:
I thought we had removed the 'v' prefix, we will bring it back?

Reply:
We did. This template is only for this repo where the `v` will actually be for better visualization when calling the workflow.
Like this:  
`groundbreaker/workflows/.github/workflows/ci-lambda.yml@v1`